lib/proofOfPatience.js

```diff
@@ -130,7 +131,7 @@ api.verifyToken = function(ip, did, token, callback) {
         }
 
         // decode and verify the token
-        token = jwt.decode(token, key.data);
+        token = jwt.verify(token, key.data, {clockTolerance: 250});
```

Comment:
We may also want to call `jwt.decode` with verify false to get the header, instead of manually doing it ourselves above.